packages/core/admin/admin/src/content-manager/components/BlocksInput/Blocks/Link.tsx

```diff
@@ -241,8 +241,8 @@ const linkBlocks: Pick<BlocksStore, 'link'> = {
         {props.children}
       </Link>
     ),
-    value: {
-      type: 'link',
+    handleConvert() {
+      // No-op, links are created via the link button in the toolbar
```

Comment:
Why even have this function? Is there a benefit to having a noop function vs only calling the function if it exists?

Reply:
I just did it to make the store's type simpler, I can make it optional if you prefer

Reply:
Maybe I'm missing something but wouldn't it just be the same as the other handlers here 🤔 :

```diff
interface NonSelectorBlock {
  renderElement: (props: RenderElementProps) => React.JSX.Element;
  matchNode: (node: Attribute.BlocksNode) => boolean;
  isInBlocksSelector: false;
- handleConvert: (editor: Editor) => void | (() => React.JSX.Element);
+ handleConvert?: (editor: Editor) => void | (() => React.JSX.Element);
  handleEnterKey?: (editor: Editor) => void;
  handleBackspaceKey?: (editor: Editor, event: React.KeyboardEvent<HTMLElement>) => void;
}
```

I think if it's a simple change it might make more sense, but it's not a huge deal. I'll leave it up to you.

Reply:
you're right it's fine, I pushed 👍